encord/client.py

```diff
@@ -587,12 +587,16 @@ def upload_image(
         else:
             raise encord.exceptions.EncordException("Image upload failed.")
 
-    def link_items(self, item_uuids: List[uuid.UUID]) -> List[DataRow]:
-        return self._querier.basic_setter(
+    def link_items(self, item_uuids: List[uuid.UUID], duplicates_behavior: str) -> List[DataRow]:
```

Comment:
Why not have this take the enum and then unpack inside the function for slightly more typesafety?

Reply:
You're totally right, I somehow misread what class this method is in.

Changed!